synapse/types/__init__.py

```diff
@@ -646,12 +646,13 @@ class StreamToken:
         7. `to_device_key`: `274711`
         8. `device_list_key`: `265584`
         9. `groups_key`: `1` (note that this key is now unused)
+        10. `un_partial_state_rooms_key`: `379`
```

Comment:
please append `_379` to the example above (line 637)

Reply:
Does this change introduce backwards/forwards incompatibilities? I'm not sure if this affects e.g. replication?

Reply:
For sync itself it's ok I believe, [a missing value will just default to 0](https://github.com/matrix-org/synapse/blob/develop/synapse/types/__init__.py#L714).

For replication streams I think they are independent of the sync token: a sync token references stream indexes, but not the other way around.